sdk/resourcemanager/README.md

```diff
@@ -33,6 +35,71 @@ If you are an existing user of the older version of Azure management library for
 ```
 [//]: # ({x-version-update-end})
 
+The services available via `azure-resourcemanager` are listed as below:
+
+<details>
+<summary> Services list </summary>
+
+- App Services
+- Authorization
+- CDN
+- Compute
+- Container Instance
+- Container Registry
+- Container Services (AKS)
+- Cosmos DB
+- DNS
+- Event Hubs
+- Insight (Monitor)
+- Key Vault
+- Managed Identity
+- Network
+- Private DNS
+- Redis
+- Resources
+- Service Bus
+- Spring Cloud
+- SQL
+- Storage
+- Traffic Manager
+</details>
+
+<br/>
+In the case where you are interested in certain service above or the service not included in the wrapper package, you can choose to use the standalone package for each service. Those packages follow the same naming patterns and design principals. For example, the package for Media Services has the following artifact information.
```

Comment:
Should we delay this section till later? My guess is most people will not need this, but all people need the azure-identity package in next section.